.changeset/red-queens-build.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/icon-button': patch
+---
+
+fix the appearance of misc theme-options
```

Comment:
Since we're introducing a new design token, we should have another changeset for that package telling about it.